.github/workflows/wheels.yml

```diff
@@ -175,9 +177,10 @@ jobs:
           fi
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@faf86a6ed7efa889faf6996aa23820831055001a  # v2.23.3
+        uses: pypa/cibuildwheel@90a0ddeff0f23eebc21630e65d66d0f4955e9b94  # v3.0.0b1
         env:
           CIBW_BUILD: ${{ matrix.python }}-${{ matrix.buildplat[1] }}
+          CIBW_ENABLE: cpython-prerelease cpython-freethreading pypy
```

Comment:
I'd set `cpython-freethreading pypy` in your pyproject.toml (or whatever toml file it's in). You can leave `CIBW_ENABLE: cpython-prerelease` here.